ui/decredmaterial/editor.go

```diff
@@ -26,6 +30,24 @@ type Editor struct {
 	shaper text.Shaper
 }
 
+type EditorCustom struct {
+	theme *Theme
+	//title is the title of the editor input field
+	title     string
+	LineColor color.RGBA
+
+	editorMaterial Editor
+	flexWidth      float32
+	editor         *widget.Editor
+
+	IsVisibleBtn     bool
+	pasteBtnMaterial IconButton
+	pasteBtnWidget   *widget.Button
+
+	clearBtMaterial IconButton
+	clearBtnWidget  *widget.Button
```

Comment:
Remove `Btn` from the names.
`IsVisibleBtn` says nothing about what this boolean control. What "is visible"?, he editor, the custom buttons?